src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -452,6 +465,11 @@ public ForgeMod()
         MinecraftForge.EVENT_BUS.addListener(this::registerPermissionNodes);
 
         ForgeRegistries.ITEMS.tags().addOptionalTagDefaults(Tags.Items.ENCHANTING_FUELS, Set.of(ForgeRegistries.ITEMS.getDelegateOrThrow(Items.LAPIS_LAZULI)));
+
+        // TODO: Remove when addAlias becomes proper API, as this should be done in the DR's above.
+        ForgeRegistry<Attribute> attrReg = (ForgeRegistry<Attribute>) ForgeRegistries.ATTRIBUTES;
+        attrReg.addAlias(new ResourceLocation("forge", "reach_distance"), new ResourceLocation("forge", "block_reach"));
```

Comment:
Pull this out into its own method, and annotate with @deprecated so that we can use static analysis to tell us when to delete it. 
`@Deprecated(forRemoval = true, since = `1.19.3`) // TODO: Remove when a proper alias API has been added to the registry, or we break world format`
Something like that.